### PR TITLE
The command to copy the decoded certificate to the server.crt

### PR DESCRIPTION
**Pull request by EmilyNamugaanyi**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.12 Features: set Milestone to 1.12 and Base Branch to release-1.12
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> Please delete this note before submitting the pull request.

### CODE REVIEW

**Comment by mistyhacks** [comment]
Deploy preview for *kubernetes-io-master-staging* ready!

Built with commit 01042229d8fcf2d3bc4810d72c7544649714cff0

https://deploy-preview-9448--kubernetes-io-master-staging.netlify.com

**Comment by jaredbhatti** [comment]
@EmilyNamugaanyi Can you please sign the CLA?

File: content/en/docs/tasks/tls/managing-tls-in-a-cluster.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ by running the following:
 
 ```console
 $ kubectl get csr my-svc.my-namespace -o jsonpath='{.status.certificate}' \
-    | base64 -d > server.crt
+    | base64 -D > server.crt
```
[the remark this answers is not in the document]
Comment (mistyhacks) [reply content/en/docs/tasks/tls/managing-tls-in-a-cluster.md]: I agree, please use the most platform-independent version of the command. Also `--decode` makes it more clear what the command is supposed to be doing.

**Comment by EmilyNamugaanyi** [comment]
@jaredbhatti I have signed it now. Am I supposed to be uploading it somewhere?

**Comment by mistyhacks** [comment]
The CLA is now signed and correct. 👍 All that is left is to address the feedback. Thanks!

**Pull request merged**